network/iptables_restore.go

```diff
@@ -0,0 +1,213 @@
+// Copyright 2015 flannel authors
```

Comment:
This probably actually belongs in `coreos/go-iptables` but having it here should be fine.

Reply:
So I leave it that way ?